Extension/src/LanguageServer/configurations.ts

```diff
@@ -132,10 +132,10 @@ export class CppProperties {
     public get ConfigurationsChanged(): vscode.Event<Configuration[]> { return this.configurationsChanged.event; }
     public get SelectionChanged(): vscode.Event<number> { return this.selectionChanged.event; }
     public get CompileCommandsChanged(): vscode.Event<string> { return this.compileCommandsChanged.event; }
-    public get Configurations(): Configuration[] { return this.configurationJson.configurations; }
+    public get Configurations(): Configuration[] { return this.configurationJson ? this.configurationJson.configurations : null; }
```

Comment:
Does this end up causing a null reference somewhere else? (as opposed to a undefined reference here)

Reply:
No.